frontend/integration-tests/tests/devconsole/template.scenario.ts

```diff
@@ -0,0 +1,55 @@
+import { browser, ExpectedConditions as until, Key, by } from 'protractor';
+import * as _ from 'lodash';
+
+import { appHost, testName, checkLogs, checkErrors } from '../../protractor.conf';
+import * as crudView from '../../views/crud.view';
+import * as secretsView from '../../views/secrets.view';
+
+const BROWSER_TIMEOUT = 15000;
```

Comment:
This timeout value should probably be an env variable.

Reply:
This is exclusively defined in many console files so I used it here